src/dotnet-svcutil/lib/src/Shared/MSBuildProj.cs

```diff
@@ -122,7 +122,14 @@ public static async Task<MSBuildProj> FromPathAsync(string filePath, ILogger log
             return project;
         }
 
-        public static async Task<MSBuildProj> ParseAsync(string projectText, string projectFullPath, ILogger logger, CancellationToken cancellationToken)
+        internal static async Task<MSBuildProj> FromPathAsync(string filePath, ILogger logger, string tfMonitor, CancellationToken cancellationToken)
```

Comment:
nit, I might rename this function to "ProjectFromPathAsync, as it returns project.  Just from reading the function name it will be clear what is intended.

Reply:
Thanks for the suggestion, it's an overloaded function, so for compatibility purpose the name remains the same as the existing one (with different number of arguments).